Base/usr/share/man/man1/cal.md

```diff
@@ -0,0 +1,77 @@
+## Name
+
+cal - Display a calendar
+
+## Synopsis
+
+```**sh
+$ cal [[[day] month] year]
+```
+
+## Description
+
+This program displays a simple calendar. If no arguments are specified, the current month is displayed with the current day highlighted.
+
+Days, months and years are specified with numbers. Week starts at Sunday.
+
+## Examples
+
+```sh
+# Display the current month
+$ cal
+    12 - 1992    
```

Comment:
Should the examples really shown the 90s? 😃

Reply:
Well, it is a bit weird if you stop to think that someone add that documentation in 2019 and wrote "the current month" as 12/1992.

However I thought that as it achieves the exact same purpose of teaching the user by example than using 12/2019 it was a nice touch to the era :)